$NetBSD: patch-ab,v 1.2 2012/12/28 03:03:08 dholland Exp $

- use standard headers
  (these functions are supposed to substitute for the libc ones, so they
  need to match stdlib.h exactly)
- declare void functions void

--- malloc.c.orig	2002-03-02 12:15:03.000000000 +0000
+++ malloc.c
@@ -45,6 +45,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <errno.h>
+#include <stdlib.h>
 #else	/* !POSIX_1 */
 #include <sys/types.h>
 extern int errno;
@@ -235,6 +236,7 @@ found:
 
 /*	freeing strategy tuned for LIFO allocation
 */
+void
 free(ap)
 register char *ap;
 {
@@ -331,6 +333,7 @@ size_t num, size;
 }
 
 /*ARGSUSED*/
+void
 cfree(p, num, size)
 char *p;
 size_t num, size;
